__tests__/app.test.js

```diff
@@ -105,3 +105,42 @@ describe("GET: 200 - /api/users", () => {
       });
   });
 });
+
+describe("PATCH: 200 - /api/articles/:articles_id", () => {
+  test("Updates votes with the required amount and returns the updated article", () => {
+    return request(app)
+      .patch("/api/articles/1")
+      .send({ inc_votes: 1 })
+      .expect(200)
+      .then(({ body }) => {
+        expect(body.article.votes).toBe(101);
+      });
+  });
+  test("400 Error if the endpoint is invalid", () => {
+    return request(app)
+      .patch("/api/articles/northcoders")
+      .send({ inc_votes: 1 })
+      .expect(400)
+      .then(({ body }) => {
+        expect(body.msg).toBe("400! Bad request!");
+      });
+  });
+  test("400 Error if the article_id is an invalid data type", () => {
```

Comment:
Very minor detail, but I think you mean inc_votes instead of article_id in this description 🙂